src/electionguard/serialize.hpp

```diff
@@ -265,17 +271,21 @@ namespace electionguard
 
     static unique_ptr<Candidate> candidateFromJson(const json &j)
     {
-        // TODO: other cases
-        if (j.contains("name") && !j["name"].is_null()) {
-            return make_unique<Candidate>(j["object_id"].get<string>(),
-                                          internationalizedTextFromJson(j["name"]),
-                                          j["party_id"].get<string>(), j["image_uri"].get<string>(),
-                                          j["is_write_in"].get<bool>());
-        }
-        auto writeIn = j.contains("is_write_in") && !j["is_write_in"].is_null()
-                         ? j["is_write_in"].get<bool>()
-                         : false;
-        return make_unique<Candidate>(j["object_id"].get<string>(), writeIn);
+        Log::debug(": deserializing");
```

Comment:
It worries me how many times strings are used in this method and are repeated in this method and this file.